src/azure-cli/azure/cli/command_modules/acr/_params.py

```diff
@@ -140,6 +148,11 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('days', type=int, help='The number of days to retain an untagged manifest after which it gets purged (Range: 0 to 365). Value "0" will delete untagged manifests immediately.', validator=validate_retention_days, default=7)
         c.argument('policy_type', options_list=['--type'], help='The type of retention policy.', arg_type=get_enum_type(RetentionType))
 
+    with self.argument_context('acr config soft-delete') as c:
+        c.argument('status', help="Indicates whether soft-delete policy is enabled.", arg_type=get_enum_type(PolicyStatus))
+        c.argument('registry_name', options_list=['--registry', '-r'])
+        c.argument('days', type=int, help='The number of days to retain a soft-deleted manifest or tag after which it gets purged (Range: 1 to 90). Default is 7.')
```

Comment:
> The number of days to retain a soft-deleted manifest or tag after which it gets purged (Range: 1 to 90). Default is 7.

At present, the range of valid values and the default value are controlled on the server side, right? Do we need to consider adding verification and default value locally?

Reply:
Right, they're validated and default set on server side. Having default set on client creates a bug during a PATCH where we don't want to change the days var but having default set resets it to 7 if the user doesn't provide a value.